content/3.workflow/3.participant-onboarding.md

```diff
@@ -12,3 +12,27 @@ To onboard the Participant, the following steps are performed:
 2. Gevamu Payments Gateway provides the Participant with the Network ID, assigns them a Participant ID, and registers the Payments CorDapp.
 3. Network ID and Participant ID are displayed in the Client Application.
 4. To access the payments service, the Participant registers with the PSP directly by submitting the combination of the Network ID and Participant ID received from the Payments Gateway.
+
+```mermaid
+sequenceDiagram
+
+  participant CA as Client Application
+  participant MW as Middleware 
+
+  box Corda Network
+  participant PN as Participant Node
+  participant GN as Gevamu Payments Gateway Node
+  end
+
+  participant PSP as Bank/PSP
+
+  CA -->> MW: Rest API
+
+  MW -->> PN: rpc call to intiate registration flow
+
+  PN ->> GN: registration flow
+  GN ->> PN: returns credentials after successful registration of node 
+
+  GN -->> PSP: Rest API
```

Comment:
![image](https://github.com/gevamu/gevamu.github.io/assets/51794590/23c4ead7-7931-43a6-8e7c-394606a1a18d)

Middleware do nothing from the side of business logic. So I think we can delete it.

It is not clear, what is the difference between solid lines and dotted ones. It would be nice to have some legend, if different lines are really needed.

One action is `REST API`, other one is `RPC call`. Name actions on the same level of abstraction. REST API is the technology to call functions. RPC call is single function call. Of course, on business abstraction level technologies are not primary.

`returns credentials after successful registration of node`. It is better to show successful registration as separated action. Also, you don't need to write `return`. In most cases calls are solid lines, returns are dotted lines.

`Rest API` - it is not clear why action exists

Reply:
I found these in a diagram, in the presentation https://docs.google.com/presentation/d/1ymv0L4OySPA13em84HrqJk96pIVYyHkg5s4iQqrPucw/edit#slide=id.g1b9204b5372_1_1